.github/workflows/test.yml

```diff
@@ -13,7 +13,7 @@ env:
   DOCKER_REGISTRY_USERNAME: ${{ github.repository_owner }}
   DOCKER_REGISTRY_TOKEN: ${{ secrets.DOCKER_REGISTRY_TOKEN }}
   DOCKER_IMAGE: ${{ github.repository }}
-  NODE_VERSION: 16
+  NODE_VERSION: 20
```

Comment:
We usually use the LTS version, and the latest is `18` 🙏

Reply:
Good to know 👍